integration-tests/cypress/commands/types.d.ts

```diff
@@ -315,7 +315,7 @@ export interface ApiRecordingSet {
   metadata?: ApiRecordingDataMetadata;
   additionalMetadata?: ApiThermalAdditionalMetadata | any;
   comment?: string;
-  processingState?: RecordingProcessingState;
+  processingState?: RecordingProcessingState | string;
```

Comment:
I think the better thing to do here is actually change the places where it's complaining from strings to the `RecordingProcessingState.XXX` enum variant, otherwise we're just throwing away the type safety.

Reply:
Yes. But then complains when I test with invalid values.  Just live with the complaint? Or can I define a singleInvalid value as a type and do RecordingProcessingState | invalidValue?

Reply:
done and cast invalid values to accepted type where needed for test